fission/function.go

```diff
@@ -497,3 +528,36 @@ func fnPods(c *cli.Context) error {
 
 	return err
 }
+
+func fnTest(c *cli.Context) error {
+	fnName := c.String("name")
+	routerURL := os.Getenv("FISSION_ROUTER")
+	if len(routerURL) == 0 {
+		fatal("Need FISSION_ROUTER set to your fission router.")
+	}
+
+	url := fmt.Sprintf("http://%s/fission-function/%s", routerURL, fnName)
+
+	fnReachable := false
+
+	resp := httpRequest(c.String("method"), url, c.String("body"), c.StringSlice("header"))
+	if resp.StatusCode < 300 {
```

Comment:
3xx are not really errors, right? I don't think we should call them errors. This should be 400...